custom/icds_reports/views.py

```diff
@@ -1717,6 +1718,9 @@ def get(self, request, *args, **kwargs):
             return JsonResponse(self.message('invalid_state'), status=400)
 
         dump = DishaDump(state_name, query_month, till_level)
+        if not request.user.is_superuser and request.GET.get('rebuild', '') == 'true':
```

Comment:
I think you've got this backwards - this prohibits superusers from rebuilding.

Also, what's the use-case here?  Who are the superusers who will be rebuilding this?  Would domain admin or some API user role be more appropriate of a check?

Reply:
i apologize. i was testing that in case the user is not a superuser it should do nothing so added the `not`.

Will we are testing this, it's just a way to rebuild without logging into the terminal and run management task. It would be removed once we would just rely on the regular task to build the files.